src/faebryk/libs/examples/buildutil.py

```diff
@@ -21,17 +22,25 @@
 from faebryk.libs.picker.picker import pick_part_recursively
 from faebryk.libs.util import ConfigFlag
 
+NAME = "example"
 BUILD_DIR = Path("./build")
-GRAPH_OUT = BUILD_DIR / Path("faebryk/graph.png")
-NETLIST_OUT = BUILD_DIR / Path("faebryk/faebryk.net")
 KICAD_SRC = BUILD_DIR / Path("kicad/source")
-PCB_FILE = KICAD_SRC / Path("example.kicad_pcb")
-PROJECT_FILE = KICAD_SRC / Path("example.kicad_pro")
 
 lcsc.BUILD_FOLDER = BUILD_DIR
 lcsc.LIB_FOLDER = BUILD_DIR / Path("kicad/libs")
 lcsc.MODEL_PATH = None
 
+build_paths: BuildPaths = BuildPaths(
+    layout=KICAD_SRC / Path(f"{NAME}.kicad_pcb"),
+    lock_file=None,
+    build=BUILD_DIR,
+    output_base=BUILD_DIR / NAME,
+    netlist=BUILD_DIR / Path("faebryk/faebryk.net"),
```

Comment:
The right side of the operator can just be a string, it doesn't need to be a `Path`
Additionally / more importantly, the strings on the RHS shouldn't contain slashes because it'll fuck w/ Windows compatibility